Angular/src/app/language-add/language-add.component.html

```diff
@@ -2,8 +2,8 @@
 	<i class="material-icons">arrow_back</i>
 </button>
 
-<h1> Add A Language </h1>
-<input class ="input-text" [(ngModel)]="languageName" placeholder="Language Name">
-<a routerLink=".." (click)="addLanguage()" title="Add Language '{{languageName}}'">
+<h1> {{ 'Add a New Language' | translate }} </h1>
+<input class ="input-text" [(ngModel)]="languageName" placeholder="{{ 'Language Name' | translate }}">
+<a routerLink=".." (click)="addLanguage()" title="{{ 'Add a New Language' | translate }} '{{languageName}}'">
```

Comment:
Will all of our translations have this same format: "{{ '*text to translate here*' | translate }}"

I want to check, because if so, it will make it easy to automate the creation and update of pot files

Reply:
As of now, yes